descope/auth/auth.go

```diff
@@ -16,6 +16,8 @@ import (
 	"golang.org/x/exp/slices"
 )
 
+const SKEW = time.Second * 5
```

Comment:
Should this be a `var` instead, so it can be changed? @aviadl

Reply:
Changing this is very sensitive, since it can cause expiration to not be caught, and this is why we didnt expose it as a standard config
But if we will see that this is needed in more usecases we will reconsider